go.mod

```diff
@@ -3,11 +3,10 @@ module github.com/tozny/e3db-clients-go
 require (
 	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/google/uuid v1.1.0
+	github.com/tozny/keycloak-client v0.0.7
```

Comment:
is this being used? It shouldn't need to as identity management service is wrapping all the Keycloak endpoints we need to call right?

Reply:
no -- it should not be used. Will double check, this should be pulled out

Reply:
Oh, I think I was in the wrong repo when I typed `go get...`